hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/DeleteMetadataActor.scala

```diff
@@ -45,12 +50,27 @@ class DeleteMetadataActor(metadataDeletionConfig: MetadataDeletionConfig) extend
       }
     case unexpected => log.warning(s"Programmer error: unexpected message $unexpected received from $sender")
   }
+
+  private def populateWorkflowsToDeleteMetadataMetric(currentTimestampMinusDelay: OffsetDateTime) = {
+    // we need the dedicated `count` query here because the result of the `queryRootWorkflowSummaryEntriesByArchiveStatusAndOlderThanTimestamp`
+    // query is upper-bounded by `batchSize` parameter, which is undesired here
+    countRootWorkflowSummaryEntriesByArchiveStatusAndOlderThanTimestamp(
+      MetadataArchiveStatus.toDatabaseValue(Archived),
+      currentTimestampMinusDelay
+    ) onComplete {
+      case Success(numToDelete) =>
+        sendGauge(workflowsToDeleteMetadataMetricPath, numToDelete.toLong, InstrumentationPrefixes.ServicesPrefix)
+      case Failure(ex) =>
+        log.warning(s"Cannot send workflowsToDeleteMetadata gauge metric value: unable to query number of workflows-to-delete-metadata from the database: ${ex.getMessage}")
```

Comment:
TOL This really shouldn't happen; perhaps `log.error` with the Throwable to get the stack trace?